src/neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -82,7 +82,7 @@ public StackItem ToStackItem(ReferenceCounter referenceCounter)
                 Abi.ToStackItem(referenceCounter),
                 new Array(referenceCounter, Permissions.Select(p => p.ToStackItem(referenceCounter))),
                 Trusts.IsWildcard ? StackItem.Null : new Array(referenceCounter, Trusts.Select(p => (StackItem)p.ToArray())),
-                Extra is null ? "null" : Extra.ToByteArray(false)
+                Extra is null ? StackItem.Null : Extra.ToByteArray(false)
```

Comment:
`null` is a valid JSON object. I think we should use `"null"`.

Reply:
But "null" it's a string value, we can't convert it again into a ContractManifest and get the same object.

Reply:
The JSON object is converted to `string`.

```
Extra.ToByteArray(false)
```

Reply:
I understand you, I added a UT in order to test it, and I will revert the change